src/components/AddUsersModal/index.jsx

```diff
@@ -0,0 +1,297 @@
+              label={
+                <React.Fragment>
+                  Email Address
+                  <span className="required">*</span>
+                </React.Fragment>
+              }
+              description="To add more than one user, enter one email address per line."
+            />
+            <p className="pb-2">
+              OR
+            </p>
+            <Field
+              id="csv-email-addresses"
+              name="csv-email-addresses"
+              component={FileInput}
+              label="Upload Email Addresses"
+              description="The file must be a CSV containing a single column of email addresses."
+              accept=".csv"
+              normalize={this.normalizeFileUpload}
+            />
+            <H3>Email Template</H3>
+            <Field
+              id="email-template-greeting"
+              name="email-template-greeting"
+              component={TextAreaAutoSize}
+              label="Customize Greeting"
+            />
+            <Field
+              id="email-template-body"
+              name="email-template-body"
+              component={TextAreaAutoSize}
+              label="Body"
+              disabled
+            />
+            <Field
+              id="email-template-closing"
+              name="email-template-closing"
+              component={TextAreaAutoSize}
+              label="Customize Closing"
+            />
+          </div>
+        </form>
+      </React.Fragment>
+    );
+  }
+
+  renderErrorMessage() {
+    const { error } = this.props;
+
+    return (
+      <div
+        ref={this.errorMessageRef}
+        tabIndex="-1"
+      >
+        <StatusAlert
+          alertType="danger"
+          iconClassName="fa fa-times-circle"
+          title="Unable to subscribe users"
+          message={error.length > 1 ? (
+            <ul className="m-0 pl-4">
+              {error.map(message => <li key={message}>{message}</li>)}
+            </ul>
+          ) : (
+            error[0]
+          )}
+        />
+      </div>
+    );
+  }
+
+  renderTitle() {
+    const { title } = this.props;
+
+    return (
+      <React.Fragment>
+        <span className="d-block">{title}</span>
+        <small>Add Subscriptions</small>
+      </React.Fragment>
+    );
+  }
+
+  render() {
+    const {
+      onClose,
+      submitting,
+      handleSubmit,
+    } = this.props;
+
+    return (
+      <React.Fragment>
+        <Modal
+          ref={this.modalRef}
+          title={this.renderTitle()}
+          body={this.renderBody()}
+          buttons={[
+            <Button
+              key="subscribe-users-submit-btn"
+              disabled={submitting}
+              className="subscribe-users-save-btn btn-primary"
+              onClick={handleSubmit(this.handleModalSubmit)}
+            >
+              <React.Fragment>
+                {submitting && <Icon className="fa fa-spinner fa-spin mr-2" />}
+                {'Assign License(s)'}
```

Comment:
Hmm 🤔 Perhaps "Add Users" so the CTA in the modal matches the action specified on the "Add Users" button that opens the modal.

Reply:
The mockups have "Assign License(s)" as the submit button text @ https://miro.com/app/board/o9J_ks6Q0ME=/ 
If mockups are not up-to-date, then let me know. I will make the change.

Reply:
Let's keep as is. Thanks!